common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -1779,6 +1784,7 @@ has_race_trait_trigger = {
 		trait = creature_dryad
 		trait = creature_frostnymph
 		trait = creature_highborne
+		trait = creature_stone_tolvir
```

Comment:
`creature_stone_tolvir` not added in `is_medium_humanoid_trigger` like `creature_tolvir` and `is_uncursed_titanforged_trigger`.